_config/pom.yml

```diff
@@ -23,23 +23,23 @@ latestFinal:
 
 # latest.version can be equal to latestFinal.version
 latest:
-  version: 7.0.0.Beta5
-  distributionZip: https://download.jboss.org/optaplanner/release/7.0.0.Beta5/optaplanner-distribution-7.0.0.Beta5.zip
-  releaseDate: 2016-12-22
+  version: 7.0.0.Beta6
+  distributionZip: https://download.jboss.org/optaplanner/release/7.0.0.Beta6/optaplanner-distribution-7.0.0.Beta6.zip
+  releaseDate: 2016-12-27
```

Comment:
Fixed with
  http://github.com/droolsjbpm/optaplanner-website/commit/328b95731
from info on docs.optaplanner.org

This wrong date caused some havoc in the upgrade recipe.